preview_generator/preview/builder/document__scribus.py

```diff
@@ -33,13 +33,15 @@ class DocumentPreviewBuilderScribus(DocumentPreviewBuilder):
     @classmethod
     def check_dependencies(cls) -> bool:
         try:
-            # BUG - 2018/09/26 - Basile - using '-v' on scribus >= 1.5 gives
-            # the version then crash, using FileNotFoundError to make the diff
-            result = check_call(['scribus', '-v'])
+            # INFO - G.M - 2019-01-17 - stderr is redirected to devnull because
+            # scribus print normal information to stderr instead of stdout.
+            result = check_call(['scribus', '-v'], stdout=DEVNULL, stderr=DEVNULL)
```

Comment:
you must put this in a log file, not to /dev/null. otherwise, in case of crash, we have no information at all.

is there some logging configured/used somewhere in preview generator?

Reply:
This code only test if scribus is present on operating system. Is there really required to have a log file for that ?

What i suggest is the Following: If `check_call` Don't raise, be ok, depency is granted. If `CalledProcessError` raise, print the command output in info or debug log. To get output in this case, see:

``` python
try:
    subprocess.check_output(...)
except subprocess.CalledProcessError as e:
    print e.output
```

@lebouquetin @tracim Do you agree ?

Reply:
@buxx @inkhey My feeling is we should have a preview-generator logger because no logging is a bad habit. This issue is the first reason for this logger, I think.

I dislike the `print` fix which will write on stdout which is often unexpected

Reply:
@lebouquetin Obviously yes, a logger. My example was only to show how get the command output in case of CalledProcessError`.

@inkhey You can implement this.

Reply:
done.